src/main/kotlin/no/nav/syfo/brev/narmesteleder/NarmesteLederBrevApi.kt

```diff
@@ -177,6 +176,11 @@ fun Route.registerNarmestelederBrevApi(
                         svarTekst = responsDTO.svarTekst,
                     )
                     if (updated) {
+                        dialogmoteService.publishNarmesteLederSvarVarselHendelse(
+                            brev = brev,
+                            responsDTO = responsDTO,
```

Comment:
Kunne man bare sendt `DialogmoteSvarType.valueOf(responsDTO.svarType)` i stedet for hele respons-objektet inn her? Så slipper vi avhengighet til `NarmesteLederResponsDTO` i lagene innover.

Reply:
Godt poeng!